src/main/java/io/kontur/disasterninja/client/EventApiClient.java

```diff
@@ -52,6 +52,7 @@ public List<EventFeedDto> getUserFeeds() {
 
     public Optional<EventApiSearchEventResponse> getEvents(String eventApiFeed, OffsetDateTime after, List<BigDecimal> bbox, int pageSize, SortOrder sortOrder) {
         String uri = String.format(EVENT_API_EVENT_LIST_URI, eventApiFeed, pageSize);
+        uri += "&episodeFilterType=NONE";
```

Comment:
I think it will be better construct url not from String but using URI type. First collect all params and then create url. May be task for refactoring for future? At least use StringBuilder for string concatenation